php/libraries/Utility.class.inc

```diff
@@ -895,34 +738,6 @@ class Utility
         return false;
     }
 
-    /**
-     * Returns the visit labels that a given test_name exists in
-     *
-     * @param string $test_name The instrument that we want to know
-     *                          which visit labels exist in.
-     *
-     * @return Array of the form array(0 => array('Visit_label' => VL))
-     *         where VL is the value returned from the database.
-     *
-     * @note    This should be moved out of the Utility class into whatever
-     *       module uses it. (Data team helper?)
-     * @note    Function comment written by Dave, not the author of this function.
-     * @cleanup
-     */
```

Comment:
IBIS redefines this function in its ProjectUtility class but neither functions are used anywhere in the code

@sruthymathew123 potential clean-up for you !